ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,187 @@
+141.7 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+145.7 "Bracing Duster / Fizzling Duster" sync / 1[56]:[^:]*:Silkie:(7764|7765):/
+146.0 "Soapsud Static?" #sync / 1[56]:[^:]*:Silkie:77ED:/
+160.1 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+166.2 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+172.3 "Eastern Ewers" sync / 1[56]:[^:]*:Silkie:776D:/
+178.6 "Rinse" sync / 1[56]:[^:]*:Eastern Ewer:776F:/ duration 7.3
+188.1 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+188.1 "Bracing Duster?" #sync / 1[56]:[^:]*:Silken Puff:7769:/
+188.1 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+195.2 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+202.3 "Carpet Beater" sync / 1[56]:[^:]*:Silkie:774F:/
+204.4 "--sync--" sync / 1[56]:[^:]*:Eastern Ewer:79FB:/
+209.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+213.4 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+225.9 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+233.0 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+233.0 "Chilling Duster?" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+233.0 "Bracing Duster?" #sync / 1[56]:[^:]*:Silken Puff:7769:/
+233.0 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+244.2 "Dust Bluster" sync / 1[56]:[^:]*:Silkie:776C:/
+249.6 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+255.7 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+262.3 "--sync--" sync / 1[56]:[^:]*:Silkie:(7751|7752):/
+263.8 "Squeaky Clean 1" sync / 1[56]:[^:]*:Silkie:7753:/
+265.5 "Squeaky Clean 2" sync / 1[56]:[^:]*:Silkie:7754:/
+266.9 "Puff and Tumble x4" #sync / 1[56]:[^:]*:Silken Puff:7771:/
+266.9 "Squeaky Clean 3" sync / 1[56]:[^:]*:Silkie:(7755|7756):/
+276.0 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+276.0 "Chilling Duster x4?" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+276.0 "Bracing Duster" #sync / 1[56]:[^:]*:Silkie:7769:/
+# 276.0 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/ # Possible here, but means a mechanic failed
+# 276.0 "Bracing Duster?" sync / 1[56]:[^:]*:Silken Puff:7769:/ # Possible here, but means a mechanic failed
+283.1 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+292.3 "Bracing Suds / Chilling Suds / Fizzling Suds" sync / 1[56]:[^:]*:Silkie:(7757|7758|7759):/
+294.4 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+299.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+303.5 "Bracing Duster / Chilling Duster / Fizzling Duster" sync / 1[56]:[^:]*:Silkie:(7764|7763|7765):/
+303.8 "Soapsud Static?" #sync / 1[56]:[^:]*:Silken Puff:77ED:/
+309.0 "Bracing Suds / Chilling Suds / Fizzling Suds" sync / 1[56]:[^:]*:Silkie:(7757|7758|7759):/
+311.1 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+316.1 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+320.0 "Bracing Duster / Chilling Duster | Fizzling Duster" sync / 1[56]:[^:]*:Silkie:(7764|7763|7765):/
+322.9 "--sync--" sync / 14:[^:]*:Silkie:79E0:/ window 15,5
+332.9 "Total Wash (enrage)" sync / 1[56]:[^:]*:Silkie:79E0:/
+338.1 "Total Wash (enrage)" sync / 1[56]:[^:]*:Silkie:79F9:/
```

Comment:
Which one of these is the enrage?

Reply:
They both are. If you survive the enrage hit somehow, such as rezzing, then 79F9 gets you after. 79F9  probably is a cycled repeat instant cast like the ones we have seen in DRS.

Reply:
I have reason to believe all bosses are like this in this instance since the instance itself does not reset, you get pulled back to the start. Also, you can get rezzed as the encounter resets too and trigger the encounter instantly.

Reply:
For what it's worth, the gladiator's second enrage cast is either 7A32 or 7A33 based of what i see for instant cast Flash of Steel. I'm not sure why there are two. Also not sure if Savage mode has these casts, oddly Show of Strength and Total Wash each only have the one version of instant cast spell and Show of Strength was added in 6.1 but used here.